R/JointModel.R

```diff
@@ -150,21 +152,36 @@ sampleStanModel.JointModel <- function(object, data, ...) {
     }
 
     args <- list(...)
+
     args[["data"]] <- append(
         as_stan_list(data),
         as_stan_list(object@parameters)
     )
 
-    if (!"init" %in% names(args)) {
-        values_initial <- initialValues(object)
-        values_sizes <- size(object@parameters)
-        values_sizes_complete <- replace_with_lookup(values_sizes, args[["data"]])
-        values_initial_expanded <- expand_initial_values(values_initial, values_sizes_complete)
-        args[["init"]] <- function() values_initial_expanded
+    args[["chains"]] <- if ("chains" %in% names(args)) {
+        args[["chains"]]
+    } else {
+        4
```

Comment:
where is this magic number coming from? could we introduce a package option for this instead?

Reply:
yeah but instead of `4` we could read this from our package option? which defaults to 4?

Reply:
and if user change does not make sense - then let's have a constant defined and documented and use the constant here